src/pages/Citywide-Stats.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <DefaultLayout>
+    <div class="citywide-stats-page">
+      <h1 class="page-title" tabindex="-1">Citywide Stats</h1>
+      <p>
+        These charts provide an overview of energy and emissions data across all
+        large buildings tracked in the city. Explore trends over time, compare
+        building performance, and identify opportunities for sustainability
+        improvements.
+      </p>
+
+      <section
+        class="graphs-container"
+        aria-label="citywide data and info for all buidings we track"
+      >
+        <article
```

Comment:
Can you change this to a `div`? It definitely is not an article! I think making this a `<section>` would make sense